.github/cloud-samples-tools/README.md

```diff
@@ -0,0 +1,89 @@
+# Cloud Samples tools
+
+This is a collection of tools used for Cloud Samples maintenance and infrastructure.
```

Comment:
issue: at the beginning of the documentation, state that the tool has two functions: `run-all` and `affected`. Describe (briefly) what each does.

Reply:
Done